.github/workflows/ci.yml

```diff
@@ -55,7 +61,7 @@ jobs:
           architecture: x64
       - uses: twisted/python-info-action@v1.0.1
       - name: Install Linux Qt5 dependencies
-        if: matrix.os.matrix == 'linux'
+        if: matrix.os.name == 'Linux'
```

Comment:
I have found in other Qt etc projects that it becomes worth being pretty explicit about the different uses of referencing each matrix item.

Reply:
Sure. Maybe that one needs to go away.